controllers/gitea_reconciler.go

```diff
@@ -181,9 +197,8 @@ func createGitUser(workshop *workshopv1.Workshop, username string, giteaURL stri
 	return reconcile.Result{}, nil
 }
 
-/**
 // Delete Gitea
-func (r *WorkshopReconciler) deleteGitea(workshop *workshopv1.Workshop, users int, giteaNamespaceName string) (reconcile.Result, error) {
+func (r *WorkshopReconciler) deleteGitea(workshop *workshopv1.Workshop, giteaNamespaceName string) (reconcile.Result, error) {
```

Comment:
@MaazMS  remove namespace name from the method! we don't need to pass it in method name

Reply:
yes